py-polars/polars/dataframe/frame.py

```diff
@@ -2394,6 +2416,11 @@ def write_ndjson(self, file: IOBase | str | Path | None = None) -> str | None:
         file
             File path to which the result should be written. If set to ``None``
             (default), the output is returned as a string instead.
+        storage_options
+            Important: this only works if file is type string
+            Extra options that make sense for ``fsspec.open()`` or a particular storage
+            connection, e.g. host, port, username, password, etc.
```

Comment:
This needs to be reworded - as a user I might not even know what fsspec is.

It would be good to include an example with `storage_options` in the Examples section.

Reply:
Fair point, I think I copied this from another read api that used this. I'll make the change there as well.